qiskit-superstaq/qiskit_superstaq/compiler_output.py

```diff
@@ -156,9 +157,18 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
     seq = None
     pulse_lists = None
 
-    if importlib.util.find_spec(
-        "qtrl"
-    ):  # pragma: no cover, b/c qtrl is not open source so it is not in qiskit-superstaq reqs
+    if "state_jp" not in json_dict:
+        warnings.warn(
+            "This output only contains compiled circuits (using a default AQT gate set). To "
+            "get back a compiled pulse sequence, you first must upload your qtrl configs using "
+            "`service.aqt_upload_configs`."
+        )
```

Comment:
nit: wording, as above
```suggestion
        warnings.warn(
            "This output only contains compiled circuits (using a default AQT gate set). To "
            "get back the corresponding pulse sequence, you must first upload your qtrl configs "
            "using `service.aqt_upload_configs`."
        )
```

Reply:
done.